wordpress-mobile/WordPress-Android
# Update login library

Brings in the latest changes to the login library.

Changes
- `Return better error messages for invalid site url in login flow`

To test:
1. Enter an invalid URL on the Self-hosted site login screen and notice the error says "The site at this address is not a WordPress site. For us to connect to it, the site must use WordPress."

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
